src/main/java/teammates/logic/core/FeedbackResponseCommentsLogic.java

```diff
@@ -67,7 +67,7 @@ public FeedbackResponseCommentAttributes createFeedbackResponseComment(
                 }
                 frComment.setId(existingComment.getId());
                 
-                return frcDb.updateFeedbackResponseComment(frComment);            
+                return frcDb.updateFeedbackResponseComment(frComment, true);            
```

Comment:
Does it impact anything if without the boolean param?
eg every time when update a comment, it will always update last edit info